src/language/printer.js

```diff
@@ -136,7 +136,7 @@ const printDocASTReducer = {
   FieldDefinition: addDescription(
     ({ name, arguments: args, type, directives }) =>
```

Comment:
@awfulaxolotl you can use default value inside destructuring:
 `({ name: arguments: args = [], directives})`

Reply:
Hey @IvanGoncharov, thanks for the reminder!

I've pushed a revised version with the cleaner syntax.